src/main/java/com/google/sps/servlets/data/util/GmapsHelper.java

```diff
@@ -0,0 +1,150 @@
+package com.google.sps.data.util;
+
+import com.google.gson.Gson;
+import com.google.maps.GaeRequestHandler;
+import com.google.maps.GeoApiContext;
+import com.google.maps.PlacesApi;
+import com.google.maps.errors.ApiException;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceType;
+import com.google.maps.model.PlacesSearchResponse;
+import com.google.maps.model.PlacesSearchResult;
+import com.google.maps.model.RankBy;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class GmapsHelper {
+
+  GeoApiContext context;
+
+  private final PlaceType[] recreationTypes = {
+    PlaceType.AMUSEMENT_PARK,
+    PlaceType.AQUARIUM,
+    PlaceType.ART_GALLERY,
+    PlaceType.BOWLING_ALLEY,
+    PlaceType.CAMPGROUND,
+    PlaceType.GYM,
+    PlaceType.MOVIE_THEATER,
+    PlaceType.MUSEUM,
+    PlaceType.NIGHT_CLUB,
+    PlaceType.PARK,
+    PlaceType.SPA,
+    PlaceType.STADIUM,
+    PlaceType.SHOPPING_MALL,
+    PlaceType.TOURIST_ATTRACTION
+  };
+
+  private final PlaceType[] diningTypes = {
+    PlaceType.BAKERY,
+    PlaceType.BAR,
+    PlaceType.CAFE,
+    PlaceType.BOWLING_ALLEY,
+    PlaceType.MEAL_DELIVERY,
+    PlaceType.MEAL_TAKEAWAY,
+    PlaceType.RESTAURANT
+  };
+
+  private final PlaceType[] groceryTypes = {
+    PlaceType.GROCERY_OR_SUPERMARKET, PlaceType.PHARMACY, PlaceType.PET_STORE, PlaceType.DRUGSTORE
+  };
+
+  public GmapsHelper(boolean useAppEngineContextBuilder) {
+
+    if (useAppEngineContextBuilder) {
+      this.context =
+          new GeoApiContext.Builder(new GaeRequestHandler.Builder())
+              .apiKey("AIzaSyCRG4mbpl4ZOXTmB8xACmqAcnnDSDdLJFU")
```

Comment:
let's use a separate file to store api key here, so it's easier to update in the future

Reply:
Will do!